README.md

```diff
@@ -3,19 +3,23 @@
 1. Create a new, empty folder called opensource/ in your computer's home folder.
 2. Navigate to it (`cd opensource`), then fork and clone the ams repo so that it gets downloaded into opensource/ams.
 3. Navigate to this folder (`cd ams`) and run `git remote add upstream https://github.com/codethechange-niec/ams.git` (or, if you're using SSH, use `git@github.com:codethechange-niec/ams.git`) to add an upstream remote that's linked to ams's main repository. (This will be necessary when submitting PRs later.)
-4. Create a new database(MySQL) by following these commands:
+4. Create a new database(MySQL) and import ams.sql to it by following these commands:
   1. `mysql -u root -p` (Enter root password when prompted)
-  2. `CREATE USER 'ams_user'@'localhost' IDENTIFIED BY 'ams@123';`
-  3. `GRANT ALL PRIVILEGES ON * . * TO 'ams_user'@'localhost';`
-  4. `exit`
-  5. `mysql -u ams_user -p`
-  6. `CREATE DATABASE ams;`
-  7. `exit`
-  8. `mysql -u ams_user -p ams < ams.sql`
+  2. `CREATE DATABASE ams;`, Do not exit mysql now if wants to create new user.
```

Comment:
This database is created under root user not ams_user.  Please fix it Create it under ams_user.

Reply:
I have done, PTAL @Hudda